code/datums/components/seethrough_mob.dm

```diff
@@ -41,6 +41,7 @@
 	var/icon/current_mob_icon = icon(fool.icon, fool.icon_state)
 	render_source_atom.pixel_x = -fool.pixel_x
 	render_source_atom.pixel_y = ((current_mob_icon.Height() - 32) * 0.5)
```

Comment:
I think I've already said this, but these lines can be moved back to `trick_mob()` (that should keep the offsets a bit more up to date until dedicated signals are implemented), sorry for the inconvenience.